docs_src/getting-started/Ch-GettingStartedUsers.md

```diff
@@ -74,6 +74,42 @@ Once you have selected the EdgeX Compose file you want to use, download it using
 !!! Note
     The commands above fetch the Docker Compose to a file named 'docker-compose.yml' in the current directory.  Docker Compose commands look for a file named 'docker-compose.yml' by default.  You can use an alternate file name but then must specify that file name when issuing Docker Compose commands.  See Compose [reference documentation](https://docs.docker.com/compose/reference/overview/) for help.  
 
+### Generate a custom docker compose file
+
+The docker compose files under the `releases/hanoi` folder contain the standard set of EdgeX services configured to use `ZeroMQ` for the message bus and include only the Device Virtual and Device REST device services. If you need to have different device services running or use `MQTT` for the message bus, you need a modified version of one of the standard docker compose files. The [`Compose Builder`](https://github.com/edgexfoundry/developer-scripts/tree/v1.3.0/compose-builder) tool was added for the Hanoi release. This tool allows users to generate custom compose files. 
```

Comment:
I am not sure this makes clear why you want to do this.  I think it might be helpful to add something like this to this paragraph...

...you need a modified version of one of the standard docker files.  You could manually add the device connectors to one of the existing EdgeX compose files or, use the EdgeX Compose Builder tool to generate new compose files to add the services you would like included.  When you use Compose Builder, you don't have to worry about adding all the necessary ports, variables, etc. as the tool will generate the service elements in the file for you.  The Compose Builder ...